Source/common/SNTKernelCommon.h

```diff
@@ -99,4 +97,10 @@ typedef struct {
   char pname[MAXPATHLEN];
 } santa_message_t;
 
+// Used for the kSantaUserClientCacheBucketCount request.
+typedef struct {
+  uint16_t per_bucket[1024];
```

Comment:
Make 1024 a constant and use throughout. Or use `sizeof(counts.per_bucket) / sizeof(uint16_t)` where you are using 1024 throughout.

Reply:
Done.